protobuf/flow/access/access.proto

```diff
@@ -49,8 +49,10 @@ service AccessAPI {
 
   // Accounts
 
-  // GetAccount gets an account by address.
+  // GetAccount gets an account by address from the latest sealed execution state
   rpc GetAccount (GetAccountRequest) returns (GetAccountResponse);
```

Comment:
```suggestion
  rpc GetAccountAtLatestBlock (GetAccountRequest) returns (GetAccountResponse);
```

How do you feel about updating this to be consistent with the script methods below?

Reply:
I thought about that, then I felt that it would be a breaking change for the NBA folks.